chains/ethereum/config/src/types.rs

```diff
@@ -1,3 +1,4 @@
+pub use rosetta_ethereum_backend::FilterBlockOption;
```

Comment:
Hmm... I don't want ethereum config depending on backend, because backend is `std` while `config` is `no-std`, will be needed to be imported in the runtime, but that ok for now, I'll refactor it later.